pynest/examples/recording_demo.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+#
+# recording_demo.py
```

Comment:
We should add this to ``doc/userdoc/examples/index.rst``.

Reply:
Good idea! Done in af7a69bfc.